polar-core/src/vm.rs

```diff
@@ -1168,6 +1178,7 @@ impl PolarVirtualMachine {
         self.log_with(
             || {
                 let mut msg = format!("LOOKUP: {}.{}", instance.to_string(), field_name);
+                // TODO (leina): add kwargs to log
```

Comment:
Seems like a good TODO to get in before shippin'

Reply:
Done.